src/parsers/pluscal.ts

```diff
@@ -91,11 +94,49 @@ export class TranspilerStdoutParser extends ProcessOutputHandler<DCollection> {
         return true;
     }
 
+    
     private addError(location: vscode.Position, message: string) {
         const locRange = new vscode.Range(location, location);
         this.result.addMessage(this.filePath, locRange, message);
     }
 
+    /**
+     * Adds info on labels added by the pluscal translated.
+     * 
+     * Only takes effect if the `-reportLabels` was added to PlusCal options. Output looks like:
+     * 
+     *       The following labels were added: // 1
+     *           Lbl_1 at line 16, column 5
+     *           Lbl_2 at line 23, column 9
+     * 
+     * 
+     * So we can start looking for labels as soon as we see (1)
+     * and stop as soon as we stop seeing label strings.
+     */
+    private tryParseAddedLabels(line: string) {
+        if (line.startsWith("The following label")) { // could be `was added` or `were added`
+            this.nowParsingAddedLabels = true;
+            return true;
+        };
+        if (!this.nowParsingAddedLabels) {
+            return false
+        }
+
+        const matcher = /^\s\s([A-Za-z0-9_]+) at line (\d+), column (\d+)$/g.exec(line)
```

Comment:
Is this robust against any accepted "label root" that a user may pass with `-labelRoot`?

Reply:
Can you please open a Pcal issue?

Reply:
Took some time but that's now done!

Reply:
https://github.com/tlaplus/tlaplus/issues/1092